modules/gltf/src/lib/extensions/KHR_draco_mesh_compression.ts

```diff
@@ -76,9 +81,24 @@ async function decompressPrimitive(
   const dracoOptions = {...options};
```

Comment:
Yes we need to do it this way. 

Unfortunately, going through `parse()` defeats the new typings. Not sure if that is fixable. We could explicitly import types from the draco loader here.
```suggestion
import {DracoLoaderOptions, DracoMeshData} from `@loaders.gl/draco';
  const dracoOptions: DracoLoaderOptions = {...options};
```